spec/models/html_document_spec.rb

```diff
@@ -213,6 +213,20 @@
 
       it { is_expected.to eq 'http://www.foo.gov/og_image.jpg' }
     end
+
+    context 'when an og:image is not available' do
```

Comment:
Please reword this to clarify what is being tested:
```ruby
context 'when og:image is not a valid URL' do
```

There should also be a spec to test that we return `nil` when there is no `og:image`.